src/model/ElectricEquipmentITEAirCooled.hpp

```diff
@@ -107,13 +109,21 @@ namespace model {
     /** @name Other */
     //@{
 
-    boost::optional<double> wattsperUnit() const;
+    /** Returns the designLevel if possible based on the underlying data of electricEquipmentITEAirCooledDefinition(). */
+    boost::optional<double> designLevel() const;
+    OS_DEPRECATED(3, 8, 0) boost::optional<double> wattsperUnit() const;
```

Comment:
That one is debatable I suppose. Lights for eg also doesn't use `designLevel` but `lightingLevel`. But I think Lights is wrong, and all SpaceLoadInstance should follow this convention.